readthedocs/core/resolver.py

```diff
@@ -63,23 +64,28 @@ def base_resolve_path(
             subproject_slug=None,
             subdomain=None,
             cname=None,
+            version_type=None
     ):
         """Resolve a with nothing smart, just filling in the blanks."""
         # Only support `/docs/project' URLs outside our normal environment. Normally
         # the path should always have a subdomain or CNAME domain
         # pylint: disable=unused-argument
-        if subdomain or cname or (self._use_subdomain()):
-            url = '/'
-        else:
-            url = '/docs/{project_slug}/'
+        if version_type != EXTERNAL:
+            if subdomain or cname or (self._use_subdomain()):
+                url = '/'
+            else:
+                url = '/docs/{project_slug}/'
 
-        if subproject_slug:
-            url += 'projects/{subproject_slug}/'
+            if subproject_slug:
+                url += 'projects/{subproject_slug}/'
 
-        if single_version:
-            url += '{filename}'
+            if single_version:
+                url += '{filename}'
+            else:
+                url += '{language}/{version_slug}/{filename}'
         else:
```

Comment:
We should keep the `if` statement smaller, not have all the existing logic inside an if. Flip the logic around, and it makes sense.